test/functional/apps/context/_size.ts

```diff
@@ -71,7 +76,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await retry.waitFor(
         `number of rows displayed after clicking load more successors is ${expectedRowLength}`,
         async function () {
-          const rows = await docTable.getRowsText();
+          // open full screen mode
+          await testSubjects.click('dataGridFullScreenButton');
+          const rows = await dataGrid.getRowsText();
+          // close full screen mode
+          await testSubjects.click('dataGridFullScreenButton');
```

Comment:
it's interesting why we need to do this, without it's flaky, right? 
Since this is a repeating pattern we could move this into the 'getRowsText' function, maybe with a argument to test is with/without fullscreen

Reply:
By default, data grid renders 9 rows. Full screen gives ability to have more rows rendered.

Reply:
thank you for the clarification